.github/workflows/test.yml

```diff
@@ -118,6 +118,29 @@ jobs:
         path: single/directory
         merge-multiple: true
 
+    - name: Request missing Artifact
+      id: request-missing-artifact
+      uses: ./
+      with:
+        name: nonexistent
+        warn-on-failure: true
+
+    - name: Check missing warning
+      env:
+        output: ${{ steps.request-missing-artifact.outputs.failure }}
+      if: ${{ !contains(env.output, 'Unable to download artifact(s):') }}
+      run: |
+        false
+      shell: bash
+
+    - name: Log missing warning
+      env:
+        output: ${{ steps.request-missing-artifact.outputs.failure }}
+        run_os: ${{ matrix.runs-on }}
+      run: |
+        echo "::notice title=This message is expected::[$run_os] $output"
+      shell: bash
```

Comment:
This demonstrates the ability for consuming workflows to take advantage of the output and report it in whichever way they see fit.

Reply:
Please attached view of demonstration if possible